src/scripts/section2.js

```diff
@@ -0,0 +1,22 @@
+const tabs = document.querySelectorAll('.section2 [data-tab-target]');
+const tabContents = document.querySelectorAll('[data-tab-content]');
+
+tabs[0].classList.add('active');
+tabContents[0].classList.add('active');
+
+tabs.forEach(tab => {
+  tab.addEventListener('click', () => {
```

Comment:
We can use event delegation instead of foreach if there many tabs to avoid writing multiple listeners.